siriuspy/siriuspy/devices/bpm.py

```diff
@@ -998,77 +1022,46 @@ def get_slow_orbit(self):
         orby = _np.array(orby)
         return orbx, orby
 
-    def get_mturn_orbit(self, return_sum=False):
-        """Get Multiturn orbit matrices.
-
-        Args:
-            return_sum (bool, optional): Whether or not to return BPMs sum.
-                Defaults to False.
+    def get_mturn_signals(self):
+        """Get Multiturn signals matrices.
 
         Returns:
-            orbx (numpy.ndarray, Nx160): Horizontal Orbit.
-            orby (numpy.ndarray, Nx160): Vertical Orbit.
-            possum (numpy.ndarray, Nx160): BPMs Sum signal.
+            list: Each component of the list is an numpy.ndarray with shape
+                (N, 160), containing the values for the signals acquired.
 
         """
-        orbx, orby = [], []
-        if return_sum:
-            possum = []
+        sigs = [[] for _ in self._mturn_signals2acq]
 
-        mini = None
+        mini = 1000000000000000  # a very large integer
```

Comment:
i like using language defined constants, instead of arbibrary number. why not `sys.maxsize`?

Reply:
done!